mediator/med_phases_prep_ocn_mod.F90

```diff
@@ -529,8 +528,10 @@ subroutine med_phases_prep_ocn_custom_cesm(gcomp, rc)
              ifracr_scaled = ifracr(n) / (frac_sum)
              ofracr_scaled = ofracr(n) / (frac_sum)
           endif
-          Foxx_swnet(n) = ofracr_scaled*(fswabsv + fswabsi) + ifrac_scaled*Fioi_swpen(n)
-
+          !TODO: ? fix this
```

Comment:
Can you provide more detail in this comment?

Reply:
I forgot to remove that TODO, but I knew that the associated change might be an issue for CESM so thanks for catching. 

I tried to make the logic for ``import_swpen_by_bands`` more similar to the logic for ``export_swnet_afracr``. So I moved the retrieval of ``Fioi_swpen`` to only happen if the band variables were not present. I couldn't think of a case where you'd have both non-banded and banded swpen from ICE, but maybe that is a use-case for CESM. The non-banded swnet is calculated only if ``import_swpen_by_bands`` is not true.